R/variable_types.R

```diff
@@ -6,29 +6,7 @@
 #'
 #' @return (atomic vector of `character`) classes of `columns` from provided `data`
```

Comment:
```suggestion
#' @return Character vector of classes of `columns` from provided `data`.
```

Also change argument descriptions:
```
#' @param data (`matrix` or `data.frame`-like) object to determine variable types from.
#' @param columns (`character`) vector of columns in `data` to get classes from.
#'   Set to `NULL` to get classes of all columns.
```

Reply:
updated here [781e30b](https://github.com/insightsengineering/teal.slice/pull/506/commits/781e30b7752b481640589d25979370c1f12fe74f)